local/configs/jenkins.yaml

```diff
@@ -107,6 +107,7 @@ unclassified:
     endpoint: "otel-collector-contrib:4317"
     useTls: false
 jobs:
+  - file: ".ci/jobs/apm-shared.yml"
```

Comment:
is this intended? If so , you might need to use full path

Reply:
Grr. that snuck in from my local dev. I'll remove it.